<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>557</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>557</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>557</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/mainwindowpic/loginpage.png);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPswText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-color:rgb(205,207,208);
border-style:solid;
border-bottom-width:2px;
font: 200 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border: 2px solid rgb(0,191,243);
font: 20px;
background:transparent;

</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerBtn_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
border: 2px solid rgb(205,207,208);
font: 20px;
background:transparent;

</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-color:rgb(205,207,208);
border-style:solid;
border-bottom-width:2px;
font: 200 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pswText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-color:rgb(205,207,208);
border-style:solid;
border-bottom-width:2px;
font: 200 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
